README.md

```diff
@@ -3,28 +3,10 @@
 ![Sync with S3](https://github.com/ember-learn/ember-api-docs-data/workflows/Sync%20with%20S3/badge.svg)
 
 The content of this repository is synced to the `s3://api-docs.emberjs.com`
-S3 bucket via a [GitHub Action](./github/workflows/sync.yml). The data is
-consumed by the [API Docs Viewer](https://github.com/ember-learn/ember-api-docs).
+S3 bucket via a [GitHub Action](./github/workflows/sync.yml). That data is
```

Comment:
this still says that we sync to S3 but we don't 🤔 and that workflow isn't there any more